react/week2/interactive-todo-list/src/components/TodoList.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+function TodoList(props) {
+  const { completeTodo, deleteTodo } = props;
+  let todoArr = props.todoArr.length > 0 ? props.todoArr : null;
+  return (
+    <div>
+      <ul>
+        {todoArr && todoArr.length > 0 ? (
+          todoArr.map((el, i) => (
+            <li key={i}>
+              <div className={el["done"] ? "strike-through" : null}>
+                {el.title}
+              </div>
+              <div>
+                <input type="checkbox" onClick={() => completeTodo(i)} />
```

Comment:
Here, we pass the `id` for both the `key` and `completeTodo`

```suggestion
          todoArr.map((el, i) => (
            <li key={el.id}>
              <div className={el["done"] ? "strike-through" : null}>
                {el.title}
              </div>
              <div>
                <input type="checkbox" onClick={() => completeTodo(el.id)} />
```